parlai/mturk/core/mturk_data_handler.py

```diff
@@ -760,5 +761,37 @@ def get_conversation_data(task_group_id, conv_id, worker_id, is_sandbox):
         result['had_worker_file'] = True
         with open(target_filename, 'r') as target_file:
             result['data'] = json.load(target_file)
-        print(result)
+
         return result
+
+    @staticmethod
+    def get_full_conversation_data(task_group_id, conv_id, is_sandbox):
+        '''Gets all conversation data saved for a world'''
+        target = 'sandbox' if is_sandbox else 'live'
+        return_data = {
+            'custom_data': {},
+            'worker_data': {},
+        }
+
+        target_dir = os.path.join(
+            data_dir, target, task_group_id, conv_id)
+        target_dir_custom = os.path.join(target_dir, 'custom')
+        custom_file = os.path.join(target_dir_custom, 'data.json')
+        if os.path.exists(custom_file):
+            custom_data = {}
+            with open(custom_file, 'r') as infile:
+                custom_data = json.load(infile)
+            pickle_file = os.path.join(target_dir_custom, 'data.pickle')
+            if os.path.exists(pickle_file):
+                with open(pickle_file, 'rb') as infile:
+                    custom_data['needs-pickle'] = pickle.load(infile)
+            return_data['custom_data'] = custom_data
+
+        target_dir_workers = os.path.join(target_dir, 'workers')
+        for w_file in os.listdir(target_dir_workers):
+            w_id = w_file.split('.json')[0]
```

Comment:
So there's a `target_dir_workers/w_id` and `target_dir_workers/w_id.json` file? what's the difference?